linera-service/src/proxy/grpc.rs

```diff
@@ -333,7 +333,7 @@ where
     /// Returns the appropriate gRPC status for the given [`ViewError`].
     fn error_to_status(err: ViewError) -> Status {
         let mut status = match &err {
-            ViewError::TooLargeValue | ViewError::BcsError(_) => {
+            ViewError::TooLargeValue | ViewError::BcsError(_) | ViewError::IncorrectRange => {
                 Status::invalid_argument(err.to_string())
             }
             ViewError::StoreError { .. }
```

Comment:
Afaict the GRPC status INVALID_ARGUMENT doesn't apply to any `ViewError`: https://grpc.io/docs/guides/status-codes/

Reply:
In fact all `ViewError`s are INTERNAL_ERROR except maybe those that will be moved to `StorageError` and capture a NOT_FOUND concept